### ISSUE TITLE
Failing Test: [sig-cluster-lifecycle] kubeadm gce, gce-selfhosting up

### ISSUE THREAD

**Issue by mohammedzee1000** [kubernetes/kubernetes#66338]
# Failing Jobs
[sig-master-blocking#kubeadm-gce](https://k8s-testgrid.appspot.com/sig-release-master-blocking#kubeadm-gce)
[sig-release-master-blocking#kubeadm-gce-selfhosting](https://k8s-testgrid.appspot.com/sig-release-master-blocking#kubeadm-gce-selfhosting)

# Failing Tests
[kubeadm gce up](https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/ci-kubernetes-e2e-kubeadm-gce/11247)
[kubeadm gce self hosting](https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/ci-kubernetes-e2e-kubeadm-gce-selfhosting/532)

# Triage Results
[kubeadm gce up](https://storage.googleapis.com/k8s-gubernator/triage/index.html?job=kubeadm-gce&test=up)
[kubeadm gce selfhosting](https://storage.googleapis.com/k8s-gubernator/triage/index.html?job=kubeadm-gce-selfhosting&test=up)

Tests are failing on sig-master-blocking

/cc @tpepper @AishSundar 
/kind bug
/priority failing-test
/priority important-soon

I am not sure of which sig this goes through to

**Comment by neolit123** [comment]
@timothysc, question.
do we want to maintain `kubeadm-gce-selfhosting` for 1.12 and in the future, given the recent selfhosting decisions?

**Comment by timothysc** [comment]
per slack, this is from the version of the test-infra file being passed into init is too old.  
I'll try to loop around to this tomorrow, unless someone else can help out.  

/cc @kubernetes/sig-cluster-lifecycle-bugs

**Comment by neolit123** [comment]
potential fixes are up:

sig-master-blocking#kubeadm-gce:
https://github.com/kubernetes/kubernetes-anywhere/pull/545

sig-release-master-blocking#kubeadm-gce-selfhosting
https://github.com/kubernetes/kubernetes-anywhere/pull/546

**Comment by neolit123** [comment]
the `v1alpha1` related errors were fixed, but now both tests are getting:
```
Jul 26 22:26:26 e2e-11347-31d72-master startup-script: INFO startup-script: + [[ ci-cross/latest -lt 11 ]]
Jul 26 22:26:26 e2e-11347-31d72-master startup-script: INFO startup-script: /startup-nxc9y49_/tmpzbkx415g: line 186: ci: unbound variable
Jul 26 22:26:26 e2e-11347-31d72-master startup-script: INFO startup-script: Return code 1.
```
for the master nodes. digging out possibly causes; if someone knows - help out. :)

**Comment by neolit123** [comment]
submitted follow-up PR:
https://github.com/kubernetes/kubernetes-anywhere/pull/549

**Comment by mohammedzee1000** [comment]
Note these tests are blocking alpha cut, whats the eta on fix.

**Comment by neolit123** [comment]
@mohammedzee1000 
waiting on https://github.com/kubernetes/test-infra/pull/8874
these are hard to debug and fix because they requires multiple steps by multiple folks.

hoping to get them working by the end of the week.

**Comment by neolit123** [comment]
`kubeadm-gce` turned green. :+1: 
waiting on a few runs for `kubeadm-gce-selfhosting`.
seems to have one good run + flake today.

**Comment by neolit123** [comment]
@mohammedzee1000 @tpepper 
we are green for both now and the fix worked. :tada: 

please note that, kubeadm-gce-selfhosting is a flaky one in general.

**Comment by mohammedzee1000** [comment]
Thanks @neolit123 
/close

**Comment by neolit123** [comment]
/reopen

tests are red again. this broke them:
https://github.com/kubernetes/kubernetes/pull/62727

working on a fix.

**Comment by mohammedzee1000** [comment]
@neolit123 We again have failures of kubeadm in gce

**Comment by mohammedzee1000** [comment]
https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/ci-kubernetes-e2e-kubeadm-gce-master/9

**Comment by neolit123** [comment]
@mohammedzee1000 yes, the SIG knows about this one.
it's probably unrelated to our work, but we are investigating.

**Comment by neolit123** [comment]
BTW, not sure if anyone else noticed but we removed `kubeadm-gce-selfhosting` as explained a couple of comments above.

**Comment by neolit123** [comment]
/close
moving the discussion to here as one test was removed and the other was renamed.
https://github.com/kubernetes/kubernetes/issues/68475

**Comment by mohammedzee1000** [comment]
/reopen
Regression https://github.com/kubernetes/kubernetes/issues/68475
